### DISCUSSION TITLE
Does stl::filesystem::copy use FILE_SHARE_READ file opening attribute?

### DISCUSSION THREAD

**Discussion by ohhmm** [microsoft/STL#1267] [General] [answered]
Thanks,
Serg

**Comment by sylveon** [comment]
It uses [CopyFile](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-copyfile)

https://github.com/microsoft/STL/blob/154f9a8a978a865bac23516d97f0824e73c25f17/stl/src/filesystem.cpp#L449

**Comment by ohhmm** [comment]
Thanks @sylveon,

Looks like https://github.com/microsoft/STL/blob/154f9a8a978a865bac23516d97f0824e73c25f17/stl/src/filesystem.cpp#L463 states an issue for update_existing?

Thanks,
Serg

**Comment by StephanTLavavej** [comment, marked as answer]
Note that LWG-3057 "Correct `copy_options` handling" remains Open.

For the Standard as currently written, we believe that our implementation is conforming. If you believe otherwise, please open an issue with a test case. Thanks!
